sigmf-spec.md

```diff
@@ -463,6 +467,54 @@ limited to no more than 20 characters because a common use is a short form GUI
 indicator. Similarly, any user interface making use of this field should be
 capable of displaying up to 20 characters.
 
+### Collection Format
+
+The `sigmf-collection` file contains JSON-formatted metadata adhering to the SigMF schema, with one top-level object called a `collection`. The `collection` object contains name/value pairs that describe relationships between SigMF Recordings.
+
+The `collection` object points to specific recordings via a _SigMF Recording tuple_, which references the base-name of the recording and the SHA512 hash of the metadata file. Tuples may be the singular value in a key/value pair, or provided in an ordered list via a JSON array.
+
+1. The `collection` object MUST be the only top-level object in the file.
+2. The `collection` object MUST only contain SigMF key/value pairs.
+3. Keys in the `collection` object MUST use SigMF Recording Tuples to reference recordings.
+4. SigMF Recording tuples MUST take the form of `["N", "hash"]`, where `N` is the base-name of a SigMF Recording and `hash` is the SHA512 hash of the Recording's metadata file `N.sigmf-meta`.
+5. If a value contains multiple SigMF Recording tuples, they MUST appear in a JSON array.  
+6. Recordings referenced in the `collection` MUST be EITHER in the immediate local directory, or in a local SigMF Archive.
```

Comment:
I interpret this two ways:

1. The `sigmf-collection` file can be in the same directory as a tar file containing the archive
2. The `sigmf-collection` file is contained within the tar file archive

If the latter (which I think you mean...) perhaps the wording here should be:

```
Recordings referenced in the `collection` MUST be EITHER in the immediate local directory, or contained within a SigMF Archive with the `collection` file.
```

Reply:
Check out my updated text and let me know what you think.

An aside: especially with collections, seems like we should support Archives in some form that is not in a tarball